src/material-experimental/mdc-slider/slider-adapter.ts

```diff
@@ -6,133 +6,179 @@
+    return this._delegate._getInput(thumb)._isFocused();
   }
-  setTrackActiveStyleProperty(propertyName: string, value: string): void {
-    throw Error('Method not implemented.');
+  getThumbKnobWidth = (thumb: Thumb): number => {
+    // TODO(wagnermaciel): Check if this causes issues for SSR
+    // once the mdc-slider is added back to the kitchen sink SSR app.
+    return this._delegate._getKnobElement(thumb).getBoundingClientRect().width;
   }
-  removeTrackActiveStyleProperty(propertyName: string): void {
-    throw Error('Method not implemented.');
+  getThumbBoundingClientRect = (thumb: Thumb): ClientRect => {
+    return this._delegate._getThumbElement(thumb).getBoundingClientRect();
   }
-  setValueIndicatorText(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  getBoundingClientRect = (): ClientRect => {
+    return this._delegate._elementRef.nativeElement.getBoundingClientRect();
   }
-  getValueToAriaValueTextFn(): ((value: number) => string) | null {
-    throw Error('Method not implemented.');
+  isRTL = (): boolean => {
+    // TODO(wagnermaciel): Actually implementing this.
+    return false;
   }
-  updateTickMarks(tickMarks: TickMark[]): void {
-    throw Error('Method not implemented.');
+  setThumbStyleProperty = (propertyName: string, value: string, thumb: Thumb): void => {
+    this._delegate._getThumbElement(thumb).style.setProperty(propertyName, value);
   }
-  setPointerCapture(pointerId: number): void {
-    throw Error('Method not implemented.');
+  removeThumbStyleProperty = (propertyName: string, thumb: Thumb): void => {
+    this._delegate._getThumbElement(thumb).style.removeProperty(propertyName);
   }
-  emitChangeEvent(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  setTrackActiveStyleProperty = (propertyName: string, value: string): void => {
+    this._delegate._trackActive.nativeElement.style.setProperty(propertyName, value);
   }
-  emitInputEvent(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  removeTrackActiveStyleProperty = (propertyName: string): void => {
+    this._delegate._trackActive.nativeElement.style.removeProperty(propertyName);
   }
-  emitDragStartEvent(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  setValueIndicatorText = (value: number, thumb: Thumb): void => {
+    this._delegate._setValueIndicatorText(value, thumb);
   }
-  emitDragEndEvent(value: number, thumb: Thumb): void {
-    throw Error('Method not implemented.');
+  getValueToAriaValueTextFn = (): ((value: number) => string) | null => {
+    return this._delegate.displayWith;
   }
-  registerEventHandler<K extends EventType>(evtType: K, handler: SpecificEventListener<K>): void {
-    throw Error('Method not implemented.');
+  updateTickMarks = (tickMarks: TickMark[]): void => {
+    this._delegate._tickMarks = tickMarks;
+    this._delegate._cdr.detectChanges();
```

Comment:
should this be `markForCheck`?

Reply:
I tried switching to `markForCheck` but it throws `ExpressionChangedAfterItHasBeenCheckedError`

Reply:
We should sit down and figure this one out- you generally _really_ want to avoid calling `detectChanges` like this. If we do conclude that it's strictly necessary, we'd need to capture why in a comment